examples/postInit/vanilla.groovy

```diff
@@ -110,6 +110,17 @@ crafting.shapedBuilder()
     .replaceByName()
     .register()
 
+// recipeFunction example
+
+crafting.shapelessBuilder()
+    .output(item('minecraft:wooden_shovel'))
+    .input(ore('logWood').reuse(),item('minecraft:wooden_shovel:*').mark('tool'))
+    .recipeFunction { output, inputs, info ->
+        def item = inputs["tool"]
```

Comment:
can you replace the double quotes with single quotes and can you comment this to describe what it does (get the marked itemstack)